examples/sdk/auth-delegated/README.md

```diff
@@ -1,225 +1,174 @@
-    signer,
-  })
-}
-```
+The exact domain association files for the tutorial project are in `./server/static`. One option is you can copy these two static files and host them under the `/.well-known` path of an internet facing server for your domain. They must be reachable through `https://[your.domain]/.well-known/`.
 
-The delegated registration is a two step process, first initialize the registration flow by invoking `createDelegatedUserRegistration` with `UserAuthKind.EndUser`.
+Another option is you can sign up for a free account on [ngrok](https://ngrok.com/). ngrok provides a tunneling service that will give you a publicly reachable domain that tunnels to a port on your `localhost`. How to setup and configure ngrok is outside the scope of this tutorial, but you can follow the instructions on their website.
 
-```typescript
-app.post(
-  '/register/init',
-  asyncHandler(async (req: Request, res: Response) => {
-    // perform local system registration before initiating Dfns registration
+Make sure after you register, create a [static domain name](https://ngrok.com/blog-post/free-static-domains-ngrok-users). Free accounts can have one static domain name, e.g. `panda-new-kit.ngrok-free.app`. Having a static domain name is necessary, otherwise when you restart a tunnel, you will get a new randomly generated domain, and the Dfns Applications you associated with the previous domain and all the Passkeys credentials on your testing devices are no longer usable.
 
-    const challenge = await apiClient().auth.createDelegatedUserRegistration({
-      body: { kind: UserAuthKind.EndUser, email: req.body.username },
-    })
-    res.json(challenge)
-  })
-)
+Once you finish setting up ngrok, you can start a tunnel to the express.js server application,
+
+```
+ngrok http --domain=panda-new-kit.ngrok-free.app http://localhost:8000
 ```
 
-A registration challenge is created along with a temporary `authToken`. They must be returned to the client-side SPA. Create a new WebAuthn credential for the end user and sign this challenge. Then invoke the second part of the registration flow with the signed challenge and the temporary `authToken`.
+For the rest of the guide, `panda-new-kit.ngrok-free.app` will be used as the example public domain name. Please replace that with the actually domain you have associated with the Dfns tutorial app.
+
+### iOS
+
+#### Prerequisites
+
+To run the React Native application on an iOS device, you must have an `Application` for iOS. To create a new `Application`, go to `Dfns Dashboard` > `Settings` > `Org Settings` > `Applications` > `New Application`, and enter the following information
+
+- Name, choose any name, for example `Dfns Tutorial iOS`
+- Application Type, leave as the default `Default Application`
+- Relying Party, set to the domain you associated with the app, e.g. `panda-new-kit.ngrok-free.app`
+- Origin, set to the full url of the domain, e.g. `https://panda-new-kit.ngrok-free.app`
```

Comment:
right now we are recommending `ios:${bundle-id}`, but I was not able to observe this behavior. I tried iOS 15 and 17, on iPhone SE and 13 Max simulators. in both cases, the origin is the full url to the associated domain. did not try on a real device.